### DISCUSSION TITLE
WYSIWYG Editor

### DISCUSSION THREAD

**Discussion by mfoitzik** [quasarframework/quasar#10321] [Show and Tell]
Hi Everyone. I built out a WYSIWYG Editor starter project using Quasar 2 and TipTap 2. You can get it at [https://github.com/mfoitzik/quasar2-tiptap2](https://github.com/mfoitzik/quasar2-tiptap2).  It is basically the examples shown in the TipTap documentation but built out in the Quasar Framework. It also adds font color and font sizing. I still have work to do on images and hyperlinks.

**Comment by pikil** [comment]
Looks nice!

**Comment by cyfung1031** [comment]
Some comments for your UI design:

![image](https://user-images.githubusercontent.com/44498510/133833045-86c87ae7-6bf0-44f6-9cea-ee6332c536ab.png)

1. The green box transition effect is somehow making users feeling bad. It's fun but not good for production use. The css animation of other boxes shall be stopped before the start of the new animation.
2. Padding is important to let users feel comfortable. You shall remove the black border or add some padding to it.

Hope you can build a good WYSIWYG Editor.

**Reply by mfoitzik** [reply]
Thank you for your comments.
